internal/restaurant/repository/menu_repo.go

```diff
@@ -59,29 +59,37 @@ func (r *RestaurantRepo) CreateMenu() (*pb.Menu, error) {
 	return menu, nil
 }
 
-func (r *RestaurantRepo) GetMenu(t time.Time) *pb.Menu {
+func (r *RestaurantRepo) GetMenu(t time.Time) (*pb.Menu, error) {
+	log.Printf("GetMenu Repository was invoked")
+	menuInst := pb.Menu{}
 	data, err := os.OpenFile("menu.json", os.O_RDONLY, 0644)
 	if err != nil {
 		log.Fatal("can't read menu.json", err)
+		return &menuInst, err
 	}
 	defer data.Close()
 	//fmt.Println("время переданное в Гетменю: ", t) //
 
-	m, _ := io.ReadAll(data)
+	m, err := io.ReadAll(data)
+	if err != nil {
+		log.Println("Can't read data from menu.json: ", err)
+		return &menuInst, err
+	}
 	menu := &pb.Menu{}
 
 	err = json.Unmarshal(m, menu)
 	if err != nil {
 		log.Fatal("cannot unmarshall menu", err)
+		return &menuInst, err
 	}

```

Comment:
вот тут сразу можно сделать довольно таки неплохую оптимизацию и лишний раз не аллоцировать память.

```suggestion
        var menu pb.Menu
        // Читаем сразу из io.Reader и пишем в menu
        if err := json.NewEncoder(data).Encode(&menu); err != nil {
            return nil, err
        }
```